packages/terra-search-field/src/SearchField.jsx

```diff
@@ -145,6 +184,9 @@ class SearchField extends React.Component {
     if (event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
       this.handleSearch();
     }
+    if (event.nativeEvent.keyCode === KeyCode.KEY_ESCAPE) {
```

Comment:
Nice, I like the addition of this keyboard interaction.

Reply:
I remember it being talked about, but later noticing that it is apparently also one of those behaviors tied to the WebKit clear button functionality, like in Chrome; like the button, hitting escape also didn’t work in Firefox.